src/main/scala/io/digitallibrary/bookapi/integration/ImageApiClient.scala

```diff
@@ -22,6 +23,40 @@ trait ImageApiClient {
 
   class ImageApiClient extends LazyLogging {
 
+    def createImage(externalId: String, filename: String, title: String, alttext: String, language: String, license: String, origin: String, author: String, imageData: Array[Byte]): Option[ImageMetaInformation] = {
```

Comment:
Treng vi opprette nytt bilde? Må ikkje en forutsetning for at du setter inn bilde i bok at bildet finnes fra før?